legend-engine-xt-documentStore-executionPlan-connection/pom.xml

```diff
@@ -71,6 +71,18 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
```

Comment:
version number?  do you want to specify in the root/pom.xml?  under dependencies??

Reply:
Good point, have add this, version already specified in pom.xml